data/card_definitions.json

```diff
@@ -6443,6 +6443,361 @@
+					"effect_type": "opponent_discard_random",
+					"amount": 1
+				}
+			}
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Bomb Burst",
+			"effects": [
+				{
+					"timing": "immediate",
+					"effect_type": "choice",
+					"choice": [
+						{ "effect_type": "advance", "amount": 3 },
+						{ "effect_type": "retreat", "amount": 3 }
+					]
+				}
+			]
+		}
+	},
+	{
+		"id": "plague_staffofsurging",
+		"type": "special",
+		"display_name": "Staff of Surging",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 1,
+		"speed": 6,
+		"power": 2,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"effect_type": "ignore_guard"
+			},
+			{
+				"timing": "after",
+				"condition": "opponent_stunned",
+				"effect_type": "draw",
+				"amount": 2
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Frantic Research",
+			"effects": [
+				{
+					"timing": "now",
+					"effect_type": "draw",
+					"amount": 2,
+					"and": {
+						"effect_type": "strike"
+					}
+				},
+				{
+					"timing": "after",
+					"effect_type": "draw",
+					"amount": 2
+				}
+			]
+		}
+	},
+	{
+		"id": "plague_longpitch",
+		"type": "special",
+		"display_name": "Long Pitch",
+		"force_cost": 1,
+		"gauge_cost": 0,
+		"range_min": 3,
+		"range_max": 5,
+		"speed": 5,
+		"power": 3,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "hit",
+				"effect_type": "force_for_effect",
+				"per_force_effect": null,
+				"overall_effect": {
+					"effect_type": "powerup",
+					"amount": 2
+				},
+				"force_max": 1
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 1,
+			"cancel_cost": -1,
+			"display_name": "Unexpected Results",
+			"effects": [
+				{
+					"timing": "now",
+					"effect_type": "choice",
+					"choice": [
+						{ "effect_type": "advance", "amount": 2 },
+						{ "effect_type": "retreat", "amount": 2 }
+					]
+				},
+				{
+					"timing": "hit",
+					"condition": "top_deck_is_normal_attack",
+					"effect_type": "add_top_deck_to_gauge",
+					"negative_condition_effect": {
+						"is_negative_effect": true,
+						"effect_type": "powerup",
+						"amount": 2,
+						"and": {
+							"effect_type": "discard_topdeck"
+						}
+					}
+				}
+			]
+		}
+	},
+	{
+		"id": "plague_chainreaction",
+		"type": "special",
+		"display_name": "Chain Reaction",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 3,
+		"speed": 5,
+		"power": 4,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "after",
+				"effect_type": "draw_to",
+				"amount": 4
+			}
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 1,
+			"cancel_cost": -1,
+			"display_name": "Bait Bomb",
+			"effects": [
+				{
+					"timing": "immediate",
+					"effect_type": "choose_cards_from_top_deck",
```

Comment:
Maybe I'm oversimplifying, but isn't this boost actually:
Look at the top 3 cards, add 1 to gauge, draw the other 2 (1 for effect 1 for end of turn)?

Reply:
I should check if there's any order enforced on it, but the situation i'm concerned about is that if there's one card left in deck they should be able to choose whether it goes to Gauge or Hand

Reply:
That's annoying.